### PR TITLE
Add a post for April 1st 2017

### PR DESCRIPTION
**Pull request by abcdefg30**
@Mailaender , @obrakmann , @pchote : I'd appreciate it if one of you could have a look at this.

filename-fullpage

If you find any nits and I'm not back in time to fix them, feel free to push over this.

### CODE REVIEW

File: content/news/april-20170401.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+The next major OpenRA release is imminent, so you might want to have a look at the upcoming features. If not, just jump directly to the [downloads](/download/) page and grab the latest playtest installer to help us getting feadback.
+
+## TibBucks
+<div style="text-align:center" markdown="1">
+![bucks](/images/news/20170401-user-tib-bucks.png)
+
+Yes, we finally managed to add the highly demanded pay2win features to OpenRA!<br>
+Buy TibBucks, our own virtual ingame currency and get your hands on our bonus content.
```
Comment (obrakmann) [inline content/news/april-20170401.md]: *premium bonus content

File: content/news/april-20170401.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<div style="text-align:center" markdown="1">
+![color_picker](/images/news/20170401-color-picker.png)
+
+The old color picker was just not intuitive enough and has been replaced<br> by this non-restrictive, user friendly gem.
+
+</div>
+
+## Brexit news: Britain also leaves the Allies
+
+No worries! We plan to readd them later as own faction ("RTG", the "Real Tough Guys"). They'll be joined by other Commonwealth members/ozeanic nations like Australia and Atlantis.
```
Comment (obrakmann) [inline content/news/april-20170401.md]: *oceanian

File: content/news/april-20170401.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+The United States of America will fill in the empty place in the Allies' rows. They got +200% build speed increase on concrete walls and also have access to Kennels and Guard Dogs. Instead of Tanya they'll get their own hero unit: Donald J. Trump himself! His special abilities are: the best. Just believe us, he has the best and most tremendous abilities.
```
Comment (obrakmann) [inline content/news/april-20170401.md]: s/place/spot/. You'll also get a cookie if you can work "bigly" into that somewhere.

**Comment by abcdefg30** [comment]
Updated.

**Pull request merged**